dataTW.py

```diff
@@ -0,0 +1,67 @@
+import tweepy as tw
+from tweepy.auth import OAuthHandler
+import csv
+import urllib2
+import re
+import os
+from dotenv import load_dotenv, find_dotenv
+from config import tweetsId, tweets_user
+
+
+load_dotenv(find_dotenv())
+
+
+CONSUMER_KEY = os.getenv('CONSUMER_KEY')
+CONSUMER_SECRET = os.getenv('CONSUMER_SECRET')
+ACCESS_TOKEN = os.getenv('ACCESS_TOKEN')
+ACCESS_SECRET = os.getenv('ACCESS_SECRET')
+
+
+auth = tw.OAuthHandler(CONSUMER_KEY, CONSUMER_SECRET)
+auth.set_access_token(ACCESS_TOKEN, ACCESS_SECRET)
+
+api = tw.API(auth)
+
+
+def get_user_ids_of_post_likes(post_id):
+    try:
+        json_data = urllib2.urlopen(
+            'https://twitter.com/i/activity/favorited_popup?id=' + str(post_id)).read()
+        found_ids = re.findall(r'data-user-id=\\"+\d+', json_data)
+        unique_ids = list(set([re.findall(r'\d+', match)[0]
+                               for match in found_ids]))
+        return unique_ids
+    except urllib2.HTTPError:
+        return False
+
+
+user_likes = get_user_ids_of_post_likes('1138112412693848065')
+users = []
+
+for tweet in tweetsId:
```

Comment:
try not to put global code like this outside functions. you can use a `main()` function as the entry point of the module (i.e. the thing that runs first and doesn't fit in any of the helper functions).